fs/xfstests.py

```diff
@@ -161,14 +167,16 @@ def tearDown(self):
             for dev in self.devices:
                 process.system('losetup -d %s' % dev, shell=True,
                                sudo=True, ignore_status=True)
-            self.part.unmount()
-
-    def _create_loop_device(self, base_disk, loop_size):
-        self.part = partition.Partition(
-            base_disk, mountpoint=self.disk_mnt)
-        self.part.mount()
-        if process.system('which mkfs.%s' % self.fs_to_test, ignore_status=True):
-            self.skip('Unknown filesystem %s' % self.fs_to_test)
+            if not self.disk_mnt == '':
+                self.part.unmount()
+
+    def _create_loop_device(self, base_disk, loop_size, mount=True):
+        if mount:
+            self.part = partition.Partition(
+                base_disk, mountpoint=self.disk_mnt)
+            self.part.mount()
+            if process.system('which mkfs.%s' % self.fs_to_test, ignore_status=True):
```

Comment:
If we can have a utility, we can change across tests later. So for now i am opting to old one.